webpack/index.d.ts

```diff
@@ -654,7 +654,20 @@ declare namespace webpack {
      */
 
     class BannerPlugin extends Plugin {
-        constructor(banner: any, options: any);
+        constructor(options?: BannerPlugin.Options | string);
+    }
+
+    namespace BannerPlugin {
+        type Filter = string | RegExp;
+        
+        interface Options {
+            banner?: string;
```

Comment:
weird, it has already used short-circuit OR to allow `options` to be undefined. but it still read the property from `options` instead of `this.options`...

I see. I' ll fix that.